client/src/components/log-in/LogInForm.js

```diff
@@ -37,11 +37,11 @@ export default function LogInForm({ handleLogInSubmit, clicked }) {
             username or password invalid. Please retry...{" "}
           </div>
           <div className="login-buttons-wrapper">
-            <button type="submit" className="generic-button">
+            <button type="submit" id="log-in" className="generic-button">
```

Comment:
If you need this `id` only for your tests, you could think about `data-testid`:
https://docs.cypress.io/guides/core-concepts/test-runner#Uniqueness